src/main/java/no/bekk/bekkopen/person/FodselsnummerCalculator.java

```diff
@@ -35,15 +35,23 @@ public static Fodselsnummer getFodselsnummerForDate(Date date){
 	 * Returns a List with with VALID Fodselsnummer instances for a given Date.
 	 *
 	 * @param date The Date instance
+	 * @param generateDNumber Whether to generate D-nummer instead of fødselsnummer
 	 * @return A List with Fodelsnummer instances
 	 */
-	public static List<Fodselsnummer> getManyFodselsnummerForDate(Date date) {
+
+	public static List<Fodselsnummer> getManyFodselsnummerForDate(Date date, boolean generateDNumber) {
```

Comment:
Jeg syns ikke vi skal eksponere noe boolsk flagg-parameter i apiet. Det skaper en type coupling jeg ikke syns vi burde ha her.
Les mer om control coupling: https://robots.thoughtbot.com/types-of-coupling#control-coupling
Se også hva Marton Fowler sier om temaet: https://www.martinfowler.com/bliki/FlagArgument.html

Det er mange måter å løse dette på. Vi kan ha en `DFodselsnummerCalculator`, vi kan ha en metode som heter `getManyDFodselsnummerForDate` på `FodselsnummerCalculator`. Vi kan også subklasse `Fodselsnummer` med `DFodselsnummer`. Mange veier til Rom. Men boolsk flagg er ikke en av dem imho.

Reply:
@eivinhb Helt ok, jeg tenkte på det etter å ha slengt inn PR. Har laget en ny gren med nytt api. Lukker denne.